Dockerfile.latest-unpriviledged

```diff
@@ -0,0 +1,19 @@
+# This container images makes the necessary modifications in the
+# inherited image (which could be OSS NGINX or NGINX Plus) in order
+# to allow running NGINX S3 Gateway as a non root user
+FROM nginx-s3-gateway
+
+# Implement changes required to run NGINX as an unprivileged user
```

Comment:
Please add a link in the comments to the official unprivileged container (https://github.com/nginxinc/docker-nginx-unprivileged/blob/main/Dockerfile-debian.template).

Reply:
Done!